packages/static-assets/scripts/get-schema.ts

```diff
@@ -229,6 +372,131 @@ export function getSchema() {
     return { elements };
 }
 
+function propFromDescription({
+    varName,
+    description,
+    arrayEntryPrefixes,
+}: {
+    varName: string;
+    description: PublicStateVariableDescription;
+    arrayEntryPrefixes: Record<string, ArrayEntryPrefixDescription>;
+}) {
+    let componentType = description.createComponentOfType;
+
+    let prop: PropertyDescription = {
+        name: varName,
+        type: componentType,
+        isArray: description.isArray,
+    };
+
+    if (description.isArray) {
+        let numDimensions = description.numDimensions || 1;
```

Comment:
I suppose `description.numDimensions` can never be zero?

Reply:
That's right, it needs to be a positive integer.